src/main/java/org/prebid/server/analytics/reporter/greenbids/model/ExtBanner.java

```diff
@@ -0,0 +1,22 @@
+package org.prebid.server.analytics.reporter.greenbids.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.iab.openrtb.request.Format;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.List;
+
+@Builder(toBuilder = true)
+@Value
+public class ExtBanner {
+
+    @JsonProperty("sizes")
+    List<Format> sizes;
```

Comment:
todo: Use a `List<List<Integer>>` to match the PBJS schema (and leverage our validation code in our API).

Reply:
fixed